### PR TITLE
Remove source lang on migrate screen also seperate removed source on alphabetical sort

### PR DESCRIPTION
**Pull request by FourTOne5**
Before:
image
After:
image

### CODE REVIEW

File: app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt [hunk 1]
```diff
@@ -21,11 +21,12 @@ import eu.kanade.tachiyomi.util.system.LocaleHelper
 fun BaseSourceItem(
     modifier: Modifier = Modifier,
     source: Source,
+    showLanguage: Boolean = true,
     onClickItem: () -> Unit = {},
     onLongClickItem: () -> Unit = {},
     icon: @Composable RowScope.(Source) -> Unit = defaultIcon,
+    content: @Composable RowScope.(Source, Boolean) -> Unit = defaultContent,
     action: @Composable RowScope.(Source) -> Unit = {},
-    content: @Composable RowScope.(Source) -> Unit = defaultContent,
```
Comment (ghostbear) [inline app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt:28]: `content` should be last. As it's the primary content for this composable and will allow to move the lambda out of parameters

**Review by ghostbear — CHANGES_REQUESTED** [review]

File: app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt (hunk 1 above)
Comment (FourTOne5) [reply app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt:28]: > and will allow to move the lambda out of parameters
How?

Comment (ghostbear) [reply app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt:28]: Kotlin allows the last lambda to be moved outside of the parameters

Before your changes 
```kt
BaseSourceItem(
  source = source
) {
  // New content
}
```

With your changes 
```kt
BaseSourceItem(
  source = source,
  content = {
    // New content
  }
) {
  // New actions
}
```

Comment (FourTOne5) [reply app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt:28]: Ahh i see

Comment (FourTOne5) [reply app/src/main/java/eu/kanade/presentation/source/components/BaseSourceItem.kt:28]: Ahh i see

**Review by ghostbear — APPROVED** [review]
I feel like this should just be an overriding of `content`, but I will allow it

**Review by arkon — APPROVED** [review]

**Comment by arkon** [comment]
Any reason why this is still a draft?

**Comment by FourTOne5** [comment]
> Any reason why this is still a draft?

To change the variable name `showLanguage` to `showLanguageInContent` cause it's only used there. But ik not on my PC now.

**Comment by FourTOne5** [comment]
Ready for merge

**Pull request merged**